modules/nodes-pods-autoscaling-creating-cpu.adoc

```diff
@@ -6,11 +6,18 @@
 
 = Creating a horizontal pod autoscaler for CPU utilization by using the CLI
 
-You can create a horizontal pod autoscaler (HPA) for an existing `Deployment`, `DeploymentConfig`, `ReplicaSet`, `ReplicationController`, or `StatefulSet` object that automatically scales the pods associated with that object to maintain the CPU usage you specify.
+Using the {product-title} CLI, you can create a horizontal pod autoscaler (HPA) to automatically scale an existing `Deployment`, `DeploymentConfig`, `ReplicaSet`, `ReplicationController`, or `StatefulSet` object. The HPA scales the pods associated with that object to maintain the CPU usage you specify.
+
+[NOTE]
+====
+It is recommended to to use a Deployment or ReplicaSet objects unless you need a 
```

Comment:
```suggestion
It is recommended to to use a `Deployment` or `ReplicaSet` object, unless you need a 
```

Reply:
Also I think "recommended to to use" is intended as "recommended not to use" as elsewhere